pegasus/src/homepage.rb

```diff
@@ -184,21 +184,28 @@ def self.get_actions(request)
         }
       ]
     elsif ["post-hoc", "pre-hoc"].include? hoc_mode
-      [
+      actions = [
         {
           text: "homepage_action_text_learn",
           type: "cta_button",
           url: CDO.studio_url("/courses"),
-        },
-        {
-          text: "homepage_action_text_codevideo",
-          type: "video",
-          youtube_id: youtube_id,
-          download_path: download_path,
-          facebook: facebook,
-          twitter: twitter
         }
       ]
+
+      code_video_action = {
+        text: "homepage_action_text_codevideo",
+        type: "video",
+        youtube_id: youtube_id,
+        download_path: download_path,
+        facebook: facebook,
+        twitter: twitter
+      }
+
+      bhm_action = {
+        type: "celebrate_bhm",
+      }
+
+      actions.push(I18n.locale == :"en-US" ? bhm_action : code_video_action)
```

Comment:
(very minor) Is locale a symbol or a string?  (In most places, I see us just compare to `en-US` but don't know which is more correct.  Either way, not too important.)

Reply:
Symbol, I tried as a string, it didn't work

Reply:
Also curious whether there's a standard of how we determine locale? I see elsewhere in that same file we look at request.locale, do you happen to know the difference?

Reply:
I do not - I'm assuming that the i18n locale is somehow set by the request locale?

Reply:
Sounds plausible :)